### ISSUE TITLE
Cannot invoke actions in quick open entries with keyboard

### ISSUE THREAD

**Issue by bpasero** [Microsoft/vscode#2493]
Still need to make sure that clicking an action with the mouse inside a quick open entry, does not close quick open (e.g. install an extension).

**Comment by bpasero** [comment]
So the issue was this: if you tab into the action to install an extension and hit enter, the action becomes disabled because the install process starts. Disabling an action causes its "tabIndex" attribute to be removed and this results in the keyboard focus to move to the root body element. This results in quick open closing because it thinks the user clicked outside of quick open.

My fix now is to not just remove the tabIndex attribute for disabled actions but actually to walk up the action hierarchy and focus the next parent node that has a tabIndex attribute. In other words, I bubble up keyboard focus instead of letting it go to root directly.

This results in quick open not closing. Instead, the tree gets focus after hitting Enter on the action to install extensions and you can use the arrow keys to navigate  to the next extension.
